cmd/kubectl-testkube/commands/dashboard.go

```diff
@@ -20,23 +27,56 @@ func NewDashboardCmd() *cobra.Command {
 			ui.Verbose = true
 			ui.Logo()
 
-			dashboardAddress := fmt.Sprintf("%slocalhost:%d/%s", DashboardURI, ApiServerPort, ApiVersion)
+			uri := fmt.Sprintf("http://localhost:%d", DashboardLocalPort)
+			if useGlobalDashboard {
+				uri = DashboardURI
+			}
+
+			dashboardAddress := fmt.Sprintf("%s/apiEndpoint?apiEndpoint=localhost:%d/%s", uri, ApiServerPort, ApiVersion)
 			ui.Success("The dashboard is accessible here:", dashboardAddress)
 			ui.Success("Port forwarding is started for the test results endpoint, hit Ctrl+C to stop")
+
+			var (
+				wg      sync.WaitGroup
+				command *exec.Cmd
+				err     error
+			)
+
+			if !useGlobalDashboard {
+				wg.Add(1)
+				go func() {
+					defer wg.Done()
+					command, err = process.ExecuteAsync("kubectl", "port-forward",
```

Comment:
let's make a func from this code, it is duplicated bellow

Reply:
hope, it looks better now